deployment/arm/deploy-region.sh

```diff
@@ -16,6 +16,9 @@ location=$4
 
 scope="$businessUnit-$appName-$env-$location"
 resourceGroupName="rg-$scope"
+# Storage account names can only contain lowercase letters and numbers
+# they need to be unique globally!
+storageActName=$(echo "${businessUnit//-}$location" | tr "[:upper:]" "[:lower:]")
```

Comment:
We should ensure that the name of the storage account is easily identifiable and includes a type like `sa` for `Storage Account` and also the business unit, location and env.  I think we're missing `env` right now.

Reply:
the SA name can only contain numbers and  lower case letters, it cant contain symbols nor upper case letters. Due to that, I minimized what goes into the name as it would be a long and unreadable name. What do you think?